ansible/roles/bootstrap-python/tasks/main.yaml

```diff
@@ -0,0 +1,53 @@
+# This can go away with ansible 2.9
+---
+
+  - name: check for python
```

Comment:
How are other Ansible tasks being run currently? I don't think we need this file.